public/app/features/variables/utils.ts

```diff
@@ -51,7 +51,7 @@ export const getSearchFilterScopedVar = (args: {
 
 export function containsVariable(...args: any[]) {
   const variableName = args[args.length - 1];
-  args[0] = isString(args[0]) ? args[0] : Object['values'](args[0]).join(' ');
+  args[0] = isString(args[0]) ? args[0] : JSON.stringify(args[0]);
```

Comment:
Two things come to mind here:

1. You should use our utility function `safeStringifyValue` to stringify this unknown value https://github.com/grafana/grafana/blob/e5e93134cff47af2076038b04947d4c963c70a9b/public/app/core/utils/explore.ts#L195
2. We are trying to limit our usage of `lodash` to methods that aren't supported out of the box. Can we replace `isString` with typeof instead?